File: eimui/src/main/java/com/xjl/eimui/messagelist/bean/EMessage.java
package com.xjl.eimui.messagelist.bean;


import java.util.HashMap;

public interface EMessage {

    String getMsgId();

    void setMine(EUser user);

    EUser getMine();

    void setOther(EUser user);

    EUser getOther();

    int getMessageType();

    String getHeaderString();

    String getContent();

    String getSubContent();

    String getMediaFilePath();

    long getDuration();

    int getProgress();

    void setProgress(int progress);

    int getMessageStatus();

    void setMessageStatus(int status);

    boolean isSelected();

    void setSelected(boolean b);

    boolean isPlaying();

    void setIsPlaying(boolean b);

    HashMap<String, String> getExtras();

}

File: eimui/src/main/java/com/xjl/eimui/messagelist/bean/EMessage.kt
package com.xjl.eimui.messagelist.bean

interface EMessage {

    val msgID : String

    var mine: EUser

    var other: EUser

    var messageType : Int

    var header: String

    var content: String?

    var subContent: String?

    var mediaFilePath: String?

    var duration: Long

    var progress: Int

    var messageStatus: Int

     var isSelected: Boolean

    var isPlaying: Boolean

    val extras: HashMap<String, String>
}
